src/MAG-Samples/KDD2020-HandsOnTutorial/README.md

```diff
@@ -0,0 +1,29 @@
+## KDD 2020 hands-on tutorial
```

Comment:
Needs header, similar to what you have in Module1 README

Reply:
added.